tests/PHPUnit/Fixtures/UITestFixture.php

```diff
@@ -57,6 +57,8 @@ public function __construct()
 
     public function setUp()
     {
+        $this->extraPluginsToLoad[] = 'CustomDirPlugin';
```

Comment:
Think you'll have to add it to top of performSetUp()

Reply:
thank you @diosmosis will try

Reply:
didn't help... https://travis-ci.org/matomo-org/matomo/jobs/489864454#L1203  will need to see tmrw and otherwise just remove all possible updates from that plugin